lib/slack/sends.ex

```diff
@@ -98,7 +98,8 @@ defmodule Slack.Sends do
       {:ok, response} ->
         case Poison.Parser.parse!(response.body, keys: :atoms) do
           %{ok: true, channel: %{id: id}} -> on_success.(id)
-          e = %{error: error_message} -> on_error.(e)
+          %{error: error_message} -> on_error.(error_message)
```

Comment:
This is changing existing behavior of passing the entire error to just passing the message. I think this line could be:

```elixir
error = %{error: _error_message} -> on_error.(error)
```